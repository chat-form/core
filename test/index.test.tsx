import React from 'react'
import Comp from '@chat-form/core'
import { render } from '@testing-library/react'

describe('Comp', () => {
  beforeAll(() => {
    jest.useFakeTimers()
  })

  afterAll(() => {
    jest.useRealTimers()
  })

  it('should render', () => {
    const { container } = render(<Comp />)
    expect(container.children).toMatchSnapshot()
  })
})
