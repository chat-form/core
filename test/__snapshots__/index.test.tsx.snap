// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comp should render 1`] = `
HTMLCollection [
  <div>
    Hello World
  </div>,
]
`;
